repository/pom.xml

```diff
@@ -258,88 +281,289 @@
+                  <override>true</override>
+                  <instructions>
+                    <Import-Package>!com.sun.*,!javafx.*,*</Import-Package>
+                  </instructions>
+                  <source>true</source>
                 </artifact>
                 <artifact>
-                  <id>org.diirt:datasource-sim:3.1.7</id>
+                  <id>org.diirt:service:3.1.8-SNAPSHOT</id>
+                  <source>true</source>
                 </artifact>
                 <artifact>
-                  <id>org.diirt:datasource-sys:3.1.7</id>
+                  <id>org.diirt:service-exec:3.1.8-SNAPSHOT</id>
+                  <source>true</source>
                 </artifact>
                 <artifact>
-                  <id>org.diirt:datasource-test:3.1.7</id>
+                  <id>org.diirt:service-jdbc:3.1.8-SNAPSHOT</id>
+                  <source>true</source>
                 </artifact>
                 <artifact>
-                  <id>org.diirt:datasource-vtype:3.1.7</id>
+                  <id>org.diirt:diirt-util:3.1.8-SNAPSHOT</id>
+                  <source>true</source>
+                </artifact>
+                <artifact>
+                  <id>org.diirt:vtype:3.1.8-SNAPSHOT</id>
+                  <source>true</source>
+                </artifact>
+                <artifact>
+                  <id>org.diirt:vtype-json:3.1.8-SNAPSHOT</id>
+                  <source>true</source>
+                </artifact>
+                <artifact>
+                  <id>org.diirt.javafx:javafx:3.1.8-SNAPSHOT</id>
+                  <override>true</override>
+                  <instructions>
+                    <Import-Package>!com.sun.*,!javafx.*,*</Import-Package>
+                  </instructions>
+                  <source>true</source>
                 </artifact>
                 <artifact>
-                  <id>org.diirt:graphene:3.1.7</id>
+                  <id>org.diirt.javafx:tools:3.1.8-SNAPSHOT</id>
+                  <override>true</override>
+                  <instructions>
+                    <Import-Package>!com.sun.*,!javafx.*,*</Import-Package>
+                  </instructions>
+                  <source>true</source>
                 </artifact>
                 <artifact>
-                  <id>org.diirt:graphene-profile:3.1.7</id>
+                  <id>org.diirt.support:diirt-ca:3.1.8-SNAPSHOT</id>
+                  <source>true</source>
                 </artifact>
                 <artifact>
-                  <id>org.diirt:service:3.1.7</id>
+                  <id>org.diirt.support:diirt-cf:3.1.8-SNAPSHOT</id>
+                  <source>true</source>
                 </artifact>
                 <artifact>
-                  <id>org.diirt:service-exec:3.1.7</id>
+                  <id>org.diirt.support:diirt-pva:3.1.8-SNAPSHOT</id>
+                  <source>true</source>
+                </artifact>
+
+                <!-- OpenJFX libraries Linux -->
```

Comment:
These are all commented out, but I thought this was where I was getting openjfx from! Where am I getting it from?

Reply:
There are two options: install openjfx from the os, or include a openjfx feature in your final product.
The org.csstudio.product product includes a openjfx feature from the efxclipse people. I have this commented out, because once the efxclipse people fix the efxclipse-rt plugin, I would like to bundle it ourselves.